internal/common/helper.go

```diff
@@ -9,6 +9,10 @@ import (
 // RandomizeDuration randomizes duration for a given variance.
 // variance is expected to be of a format 0.1 for 10%, 0.5 for 50% and so on
 func RandomizeDuration(variance float64, duration time.Duration) time.Duration {
+	// do not allow less than a second requeue
+	if duration.Milliseconds() < 1000 {
```

Comment:
In the old code, how was the duration ever anything other than `time.Second * 5`? I'm not understanding how the original problem was occurring.

Reply:
discussed in person.